vita3k/renderer/src/gl/renderer.cpp

```diff
@@ -128,7 +128,7 @@ static void before_callback(const char *name, void *funcptr, int len_args, ...)
 #endif // MICROPROFILE_ENABLED
```

Comment:
microprofile remainings

Reply:
Yes, Microprofile references were left in some cases as a reminder to help keeping track of parts of the emulator that were once profiled with Microprofile but that haven't been implemented yet with Tracy, specially in the renderer and the shader recompiler of the emulator due to complexity. Since Tracy has advanced integration with OpenGL and Vulkan being capable of displaying frame buffers and other features, using regular profiling in the renderer wouldn't make much sense so I left the Microprofile code in purpose to keep which functions and parts of the renderer were profiled before so that they can be re-implemented at a later date either by me or by any other dev using the integration features that Tracy has for the graphics API.